calculator.rb

```diff
@@ -0,0 +1,142 @@
+# The program should use the input operation and two numbers
+# to provide the result of applying the operation to the two numbers
+
+# The program should have support for these four operations: addition,
+# subtraction, multiplication, and division
+
+# The program should accept both the name (add) and the symbol
+# (+) for each possible operation
+
+# The program should handle unexpected user input
+
+# Print operators options method
+def print_operator_options()
+  puts "1. add(+)"
+  puts "2. subtract(-)"
+  puts "3. multiply(*)"
+  puts "4. divide(/)"
+  puts "5. modulo(%)"
+  puts "6. exponential(^)"
+  puts
+end
+
+
+# Addition method
+def add_numbers(num1, num2)
+  return num1 + num2
+end
```

Comment:
I like that you've broken each of these operations out as its own method, and that you're returning a value rather than `puts`ing it here.